pkg/trace/stats/concentrator.go

```diff
@@ -32,19 +33,18 @@ type Concentrator struct {
 	// wait such time before flushing the stats.
 	// This only applies to past buckets. Stats buckets in the future are allowed with no restriction.
 	bufferLen int
-
-	In  chan []Input
-	Out chan []Bucket
-
-	exit   chan struct{}
-	exitWG *sync.WaitGroup
-
-	buckets map[int64]*RawBucket // buckets used to aggregate stats per timestamp
-	mu      sync.Mutex
+	In        chan []Input
+	Out       chan []*pb.ClientStatsPayload
+	exit      chan struct{}
+	exitWG    *sync.WaitGroup
```

Comment:
I know this isn't your work but if you removed this pointer it would be so nice :) It's not even necessary to initialise waitgroups